packages/svelte-view3d/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@egjs/svelte-view3d",
+  "version": "2.0.0",
+  "description": "Fast & Customizable glTF 3D model viewer, packed with full of features!",
+  "main": "dist/view3d.cjs.js",
+  "module": "dist/view3d.esm.js",
+  "sideEffects": [
+    "**/*.css",
+    "**/*.sass"
+  ],
+  "es2015": "dist/view3d.esm.js",
+  "types": "declaration/index.d.ts",
+  "svelte": "lib/index.js",
```

Comment:
It would be better to change the build result to *.svelte extension.

In the build of the service, it seems that *.js can be passed in the transpilation target.

Reply:
I guess that's okay, as `index.js` and `View3D.js` is just there for linking files.
They're actually pointing actual View3D.svelte file.

```js
/* eslint-disable @typescript-eslint/naming-convention */
import View3D from "./View3D";
export * from "@egjs/view3d";
export default View3D;
```

> index.js

```js
import { withMethods } from "@egjs/view3d";

import View3D from "./View3D.svelte";

withMethods(View3D.prototype, "view3D");

export default View3D;
```

> View3D.js